aws/rand-password/main.tf

```diff
@@ -0,0 +1,49 @@
+
+locals {
+  password_path = "/${var.environment}/${var.service_name}/${var.keyname}"
+
+  password_defaults = {
+    length      = "24"
```

Comment:
Would be nice to have these keys in the readme, or maybe just a link to the dict?  I assume special/upper/lower are just pass throughs for the random_password provider, so maybe just a link to their docs would work?

Reply:
Added readme in 364b7130853fb808acaf931d9d5c69db61621d0f